docs/download/index.md

```diff
@@ -11,7 +11,7 @@ By installing you agree to the [Omnissa general terms](https://www.omnissa.com/g
 
 | Horizon Server version                                                                                                         | Download Powercli module |
 |------------------------------------------------------------------------------------------------------------------------| --- |
-| 8.13/2406                    | [horizon4powercli_2406](https://artifactory.air-watch.com:443/artifactory/github-publish-local/euc-vdi/horizonpowercli/build-horizon4powercli/103/build/1/horizon4powercli.zip) |
+| 8.13/2406                    | [horizon4powercli_2406](zip/horizon4powercli_2406.zip) |
```

Comment:
Hmm, this is something we want to explicitly avoid doing (i.e. upload build artifacts to the docs repos).

Instead, we would prefer to upload the build artifacts/binaries by creating a release.

Here's the gh CLI command, I used to create a release for this zip file:

```
gh release create 24.06.0 \
    --title 24.06.0 \
    --notes "v24.06 version of the PowerCLI module for Horizon View." \
    docs/download/zip/horizon4powercli_2406.zip
```

Now, we can change this link to point to the asset in the release:

```suggestion
| 8.13/2406                    | [horizon4powercli_2406](https://github.com/euc-dev/horizon-powercli/releases/download/24.06.0/horizon4powercli_2406.zip) |
```

Reply:
Thanks, have updated the link though not sure if that is the right way to do it. Please let me know if this is not the correct way.